src/sentry/static/sentry/app/views/inviteMember/inviteMember.jsx

```diff
@@ -107,8 +108,7 @@ const InviteMember = React.createClass({
     let {email} = this.state;
     let emails = this.splitEmails(email);
     if (!emails.length) return;
-    this.setState({loading: true});
-
+    this.setState({busy: true});
```

Comment:
`busy` is now used only for submit button
`loading` corresponds to the view loading state